ci-operator/step-registry/hypershift/mce/dump/hypershift-mce-dump-commands.sh

```diff
@@ -20,11 +20,12 @@ else
   echo "MCE version is greater than or equal to 2.4, need to extract HyperShift cli"
   oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
   HO_IMAGE=$(oc get deployment -n hypershift operator -ojsonpath='{.spec.template.spec.containers[*].image}')
-  oc image extract "${HO_IMAGE}" --path /usr/bin/hypershift:/tmp --registry-config=/tmp/.dockerconfigjson
+  mkdir /tmp/hs-cli
+  oc image extract "${HO_IMAGE}" --path /usr/bin/hypershift:/tmp/hs-cli --registry-config=/tmp/.dockerconfigjson
```

Comment:
were you intended to extract the `hypershift` CLI, or the productized `hcp` CLI?

Reply:
the hypershift cli. that's the only command that has the `dump` action in it. the hcp does not.